CardanoSharp.Wallet/Models/Transactions/Transaction.cs

```diff
@@ -5,16 +5,17 @@
 namespace CardanoSharp.Wallet.Models.Transactions
 {
     /// <summary>
-    /// Parent class to create transactions. If a child class does not have 
-    /// a comment description, it means that it is a Rust CBOR libary specific 
-    /// type, and type Rust type will match the C# type exactly and it would be 
-    /// repetitive to put the comment. 
+    /// Parent class to create transactions. If a child class does not have
+    /// a comment description, it means that it is a Rust CBOR libary specific
+    /// type, and type Rust type will match the C# type exactly and it would be
+    /// repetitive to put the comment.
     /// Rust libary used for reference -> https://github.com/Emurgo/cardano-serialization-lib/blob/master/rust/src/lib.rs
     /// </summary>
     public class Transaction
     {
         public TransactionBody TransactionBody { get; set; }
         public TransactionWitnessSet TransactionWitnessSet { get; set; }
+        public bool IsValid { get; set; } = true;
```

Comment:
Interestingly. It looks like the Cardano Serialization Lib sets this to true and doesn't do anything with it for the rest of the library:

https://github.com/Emurgo/cardano-serialization-lib/blob/3c85d18d80fc6692da5b1d8e0eb5be3f563329ac/rust/src/lib.rs#L143